### DISCUSSION TITLE
Setting up iframe pane in v3

### DISCUSSION THREAD

**Discussion by simonpeters** [sanity-io/sanity#3859] [Sanity Studio Support]
Has anyone figured out how to implement iframe pane with V3?

On the readme of https://github.com/sanity-io/sanity-plugin-iframe-pane it still uses `./src/deskStructure.js`

But in the docs of V3 we use a different method. https://beta.sanity.io/docs/platform/studio/v2-to-v3/prod-url-resolver

Cheers

**Comment by pixeleate** [comment]
@simonpeters I have opened a PR with the updated instructions to use it with the V3 https://github.com/sanity-io/sanity-plugin-iframe-pane/pull/12
